mappings/net/minecraft/entity/boss/dragon/EnderDragonEntity.mapping

```diff
@@ -2,6 +2,9 @@ CLASS net/minecraft/class_1510 net/minecraft/entity/boss/dragon/EnderDragonEntit
 	FIELD field_18120 CLOSE_PLAYER_PREDICATE Lnet/minecraft/class_4051;
 	FIELD field_20865 yawAcceleration F
 	FIELD field_30428 MAX_HEALTH I
+	FIELD field_30430 TAKEOFF_THRESHOLD F
```

Comment:
Might document this, like this is a percentage of full health, and once a dragon takes this much damage while sitting it will takeoff